plugins/modules/zos_backup_restore.py

```diff
@@ -282,8 +282,8 @@
     space: 1
     space_type: g
 
-- name: Restore data sets from backup stored in the UNIX file /tmp/temp_backup.dzp.
-    Use z/OS username as new HLQ.
+- name: Restore data sets from a backup stored in the UNIX file /tmp/temp_backup.dzp.
```

Comment:
Shouldn't the behavior be that you restore data sets with ever is configure for `hlq`?

Something like:
```
Restore the data sets with the HLQ configured for option C(hlq), if the option was not set, the original HQL is used for restoration. 
```

Reply:
i think the point here is to highlight two things, firstly that backups can be stored in USS,  secondly, since no hlq is set, the default behavior kicks in (hence the 2nd sentence in the description)
> Restore the data sets with the original high level qualifiers.

Reply:
I agree with @ketankelkar